src/app/clark.component.ts

```diff
@@ -1,9 +1,14 @@
-import { Router } from '@angular/router';
+import { Router , NavigationEnd, ActivatedRoute} from '@angular/router';
 import { Component, OnInit } from '@angular/core';
 import { AuthService } from './core/auth.service';
 import { CartV2Service } from './core/cartv2.service';
 import { ModalService, ModalListElement } from './shared/modals';
 import { trigger, transition, style, animate } from '@angular/animations';
+import { Title } from '@angular/platform-browser';
+import 'rxjs/add/operator/filter';
+import 'rxjs/add/operator/map';
```

Comment:
Are we using the map and mergeMap imports? If not, we should remove the imports.